lua-linter/scripts/install-lua-language-server.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+CACHE_DIR="${HOME}/.cache/lua-language-server"
+
+# Check if cached version exists
+if [[ -x "${CACHE_DIR}/bin/lua-language-server" ]]; then
+  echo "lua-language-server already present in cache"
+  "${CACHE_DIR}/bin/lua-language-server" --version
+  exit 0
+fi
+
+# Resolve latest version via GitHub's redirect
+VERSION=$(
+  curl --fail --silent --head --location "https://github.com/LuaLS/lua-language-server/releases/latest" \
+    | awk --field-separator '/' '/^location:/ {print $NF}' \
+    | tr --delete '\r'
+)
+
+if [[ -z "${VERSION}" ]]; then
+  echo "Error: failed to resolve latest lua-language-server version" >&2
+  exit 1
+fi
```

Comment:
As specified in my previous review, we don't care to resolve the latest version. GitHub provide links to download the latest version, whatever it is